ReSharper.FSharp/src/FSharp.Psi.Features/src/QuickFixes/ConvertTupleToArrayOrListElementsFix.fs

```diff
@@ -0,0 +1,43 @@
+﻿namespace JetBrains.ReSharper.Plugins.FSharp.Psi.Features.Daemon.QuickFixes
+
+open System.Text.RegularExpressions
+open JetBrains.ReSharper.Plugins.FSharp.Psi
+open JetBrains.ReSharper.Plugins.FSharp.Psi.Features.Daemon.Highlightings
+open JetBrains.ReSharper.Plugins.FSharp.Psi.Features.Daemon.QuickFixes
+open JetBrains.ReSharper.Plugins.FSharp.Psi.Impl
+open JetBrains.ReSharper.Plugins.FSharp.Psi.Parsing
+open JetBrains.ReSharper.Plugins.FSharp.Psi.Tree
+open JetBrains.ReSharper.Psi.ExtensionsAPI
+open JetBrains.ReSharper.Psi.ExtensionsAPI.Tree
+open JetBrains.ReSharper.Psi.Util
+open JetBrains.ReSharper.Resources.Shell
+
+type ConvertTupleToArrayOrListElementsFix(expr: IFSharpExpression, actualType: string) =
+    inherit FSharpQuickFixBase()
+
+    let regex = Regex("'\w\d*( \* '\w\d*)+")
+    let arrayOrListExpr = ArrayOrListExprNavigator.GetByExpression(expr.IgnoreParentParens())
+    let isArray = arrayOrListExpr :? IArrayExpr
+
+    new (warning: AddTypeEquationError) =
```

Comment:
Any reason the second constructor is needed?

Reply:
It is planned to add support for other warnings in the future. But okay, this constructor can be removed now.